src/OrganisationRegistry.Api/Backoffice/Organisation/BankAccount/OrganisationBankAccountCommandController.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace OrganisationRegistry.Api.Backoffice.Organisation.BankAccount;
+
+using System;
+using System.Threading.Tasks;
+using Infrastructure;
+using Infrastructure.Security;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using OrganisationRegistry.Infrastructure.Commands;
+using OrganisationRegistry.Organisation;
+
+[ApiVersion("1.0")]
+[AdvertiseApiVersions("1.0")]
+[OrganisationRegistryRoute("organisations/{organisationId}/bankAccounts")]
+[OrganisationRegistryAuthorize]
+public class OrganisationBankAccountCommandController : OrganisationRegistryController
```

Comment:
Just an idea: would it make sense to make 2 types of `OrganisationRegistryController`? One for reads, without `CommandSender`, the other for commands, with `CommandSender` and `OrganisationRegistryAuthorize` on it by default? @QuintenGreenstack @W0dan